### PR TITLE
Mobile  & Desktop version

### PR DESCRIPTION
**Pull request by chiztechnology**
Hello these are some interesting featured developped in my capstone 1 project.
Please have a look on the code and on functionnalities and let me know if there's something that require changes or if I can merge it to my main branch.

Faithfully yours 🎊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,164 @@
...
+
+> Some features of the project
+
+- **[Home page | (program & Speakers]**
+- **[About page]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
```
Comment (OLIPLICHE) [inline README.md:74]: - Make sure to record a video presenting the features of the project. The maximum length of the video needs to be 5 minutes. Also, highlight a piece of the code that you find interesting or are proud of making.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,164 @@
...
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
Comment (OLIPLICHE) [inline README.md:150]: - Make sure the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                </li>
+                <li class="wide-nav-li">
+                    <a href="#featured-speaker-container" class="wide-nav-link">Speakers</a>
+                </li>
+            </ul>
+        </nav>
+        <!-- hamburger -->
+        <nav class="nav-section">
+            <a href="#" id="hamburger"><img src="img/hamburger.png" alt="menu icon" class="hamburger-img" /></a>
+        </nav>
```
Comment (OLIPLICHE) [inline index.html:55]: - **Mobile Size**:
- Make sure It contains a hamburger menu on the left side.
- Its background is transparent or the same color/pattern as the top section background.
- See an Example from the original design:
- 
- 
Capture
-
- See your design:
- 
Capture

File: index.html [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+                    <a href="about.html" class="wide-nav-link" >About</a>
+                </li>
+                <li class="wide-nav-li">
+                    <a href="#main-program-container" class="wide-nav-link" >Program</a>
+                </li>
+                <li class="wide-nav-li">
+                    <a href="#featured-speaker-container" class="wide-nav-link">Speakers</a>
+                </li>
+            </ul>
+        </nav>
```
Comment (OLIPLICHE) [inline index.html]: # **Desktop Size**: Make sure that
- It is divided in a top dark bar and a bottom white bar.
- The top short bar section with social media links, log out, and language links aligned to the right.
- The taller bottom bar with a logo and a list of page links.
- 
Capture
- 
- **Your design**
Capture
# NB:
- **About Page: It has a dark background**.
- 
Capture1

- **Home Page: It has a white background.**
- 
Capture

File: index.html [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+                <img class="partner-logo" src="img/naver_logo.png" alt="Naver logo" />
+                <img class="partner-logo" src="img/daumkakao_logo.png" alt="DaumKakao logo" />
+                <img class="partner-logo" src="img/mozilla_logo.png" alt="Mozilla logo" />
+            </div>
+
+        </div>
+
+    </main>
+
+    <footer>
```
Comment (OLIPLICHE) [inline index.html:163]: - **Footer: Mobile Size**
- Please work on the height of the footer.
- **See your design**
- 
Capture
- **Original Design**
- 
Capture

File: index.html [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+                </article>
+            </div>
+            <button class="button-join-submit">Join CC Global Summit 2022</button>
+        </div>
+        <!-- featured speakers -->
+        <div class="featured-speaker-container" id="featured-speaker-container">
+            <h2 class="featured-speaker-title">Featured Speakers</h2>
+            <div class="speaker-articles-container" id="speaker-articles-container">
+                <!-- dynamic data -->
+            </div>
```
Comment (OLIPLICHE) [inline index.html:146]: - **Please provide six (6) individual articles, Not four (4)**
- 
Capture
- **Your Design**:
- 
Capture

File: about.html [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+        <title>About page | Africa Summit Conference</title>
+        <meta http-equiv="" content="" />
+        <meta http-equiv="content-type" content="text/html;charset=UTF-8" />
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" href="css/styles.css" type="text/css" media="all" />
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;400;700&display=swap" rel="stylesheet">
+    </head>
+    <body>
```
Comment (OLIPLICHE) [inline about.html]: # Mobile Size
- The middle section article list is organized in a vertical column.
- The middle section individual articles have their elements organized in an horizontal row, with the image at the left, title in the center, and description at the right.
- The bottom section articles are organized in a vertical column.
- **Your Design**:
- 
Capture
- **NB: Please open this link to refer to the[ Original Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)**

File: about.html [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+                                <img src="img/icon_next.png" alt="" class="mobile-nav-icon-next">
+                            </div>
+                        </a>
+                    </li>
+                </ul>
+            </nav>
+    
+        </header>
+        
+        <main>
```
Comment (OLIPLICHE) [inline about.html]: # About Page
- It contains a top section with a centered column that includes a page title, a text block with the page description, and a line with a contact information or any other prompt under it.
- 
Capture1
- **Please follow the original design**
- 
Capture

File: about.html [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+                <div class="article-holder">
+                    <article class="past-cc-article cc-article-buenos">
+                        <h2 class="past-cc-article-date">2013</h2>
+                        <h4 class="past-cc-article-desc">CC Global Summit in Buenos Aires</h4>
+                    </article>
+                     <article class="past-cc-article cc-article-warsaw">
+                        <h2 class="past-cc-article-date">2011</h2>
+                        <h4 class="past-cc-article-desc">CC Global Summit in Warsaw</h4>
+                    </article>
+                </div>
```
Comment (OLIPLICHE) [inline about.html]: - NB: Please, the Section tag `<section>` defines the section of documents such as chapters, headers, footers or any other sections. Kindly make use of it in About and index HTML.
- Use this link to learn more about[ Semantic HTML ](https://www.w3schools.com/html/html5_semantic_elements.asp)

File: index.html [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+            <h2 class="welcome-subtitle">Hello Sharing World</h2>
+            <h1 class="welcome-title">CREATIVE COMMONS GLOBAL SUMMT 2022</h1>
+            <div class="welcome-description-container">
+                <p class="welcome-description-text">
+                    A joyfull celebration believing in the value of openness and sharing.
+                    creating a positive change with people from over 80 place taking place in October in Korea.
+                </p>
+            </div>
+            <div class="welcome-date-adress-container">
+                <h1 class="welcome-date-txt">2015.10.15(THU) ~ 16(FRI)</h1>
```
Comment (OLIPLICHE) [inline index.html]: - **Please, Here's how Hn tags should be used:** For an article or webpage, remember that the H1 title is the most important section. H2 and H3 are used to organize sub-sections, while H4, H5, and H6 are intended to provide additional information, with more details.
- Kindly Fix the use of `h1` in both inde.html and About.Html

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @chiztechnology, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit has a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- GitHub flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️
- Please follow the original DESIGN as indicated in the requirement of this Project.
- [Here is the link to the Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @chiztechnology    👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**